.github/workflows/centos7.yml

```diff
@@ -0,0 +1,103 @@
+name: test_apisix_on_centos7
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:     
+  test_apisix:
+    name: test_apisix_on_centos7
+    runs-on: ubuntu-latest
+
+    services:
+      etcd:
+        image: bitnami/etcd:3.4.0
+        ports:
+          - 2379:2379
+          - 2380:2380
+        env:
+          ALLOW_NONE_AUTHENTICATION: yes
+          ETCD_ADVERTISE_CLIENT_URLS: http://0.0.0.0:2379
+
+      old_etcd:
+        image: bitnami/etcd:3.3.8
+        ports:
+          - 3379:2379
+          - 3380:2380
+        env:
+          ALLOW_NONE_AUTHENTICATION: yes
+          ETCD_ADVERTISE_CLIENT_URLS: http://0.0.0.0:2379
+    
+    steps:
+
+    - name: Install Redis Cluster
+      uses: vishnudxb/redis-cluster@1.0.5
+      with:
+        master1-port: 5000
+        master2-port: 5001
+        master3-port: 5002
+        slave1-port: 5003
+        slave2-port: 5004
+        slave3-port: 5005
+
+    - name: Running Redis Cluster Test
+      run: |
+        sudo apt-get install -y redis-tools
+        docker ps -a
+        redis-cli -h 127.0.0.1 -p 5000 ping
+        redis-cli -h 127.0.0.1 -p 5000 cluster nodes
+
+    - name: Running etcd server with TLS
+      run: |
+        sudo docker run -d -p 12379:12379 -p 12380:12380 \
+        -e ALLOW_NONE_AUTHENTICATION=yes \
+        -e ETCD_ADVERTISE_CLIENT_URLS=https://0.0.0.0:12379 \
+        -e ETCD_LISTEN_CLIENT_URLS=https://0.0.0.0:12379 \
+        -e ETCD_CERT_FILE=/certs/etcd.pem \
+        -e ETCD_KEY_FILE=/certs/etcd.key \
+        -e GITHUB_ACTIONS=true \
+        -e CI=true \
+        -v /apisix/t/certs:/certs \
+        bitnami/etcd:3.4.0
+
+    - name: run centos7 docker and install dependencies 
+      run: |
+        docker pull centos:7
+        docker run -itd -v /home/runner/work/apisix/apisix:/tmp/apisix --name centos7Instance --net="host" --restart=always docker.io/centos:7 /bin/bash
```

Comment:
`--restart=always` we can remove this argument

Reply:
fixed.